pkg/controller/factory/builder.go

```diff
@@ -77,6 +77,10 @@ func BuildInstanceSet(synthesizedComp *component.SynthesizedComponent, component
 		SetReplicas(synthesizedComp.Replicas).
 		SetMinReadySeconds(synthesizedComp.MinReadySeconds)
 
+	if shardingName, ok := synthesizedComp.Labels[constant.KBAppShardingNameLabelKey]; ok {
```

Comment:
Pass labels to underlying resources via spec.labels.

Reply:
restart ops will list the instanseSet list and set the annotations to restart them, and now instanceSet misses the sharding label.  What does spec.labels mean?

Reply:
component.spec.labels.